src/screens/TokenDetail.js

```diff
@@ -74,13 +74,16 @@ class TokenDetail extends React.Component {
    * Fetch NFT info for custom NFT Tokens
    */
   fetchNftList() {
-    fetch(helpers.nftTokenListUrl()).then((response) => {
-      response.json().then((data) => {
-        this.setState({
-          nft: data.find(obj => obj.id === this.state.paramUID)
+    const nftTokenListUrl = helpers.nftTokenListUrl();
+    if (nftTokenListUrl) {
+      fetch(helpers.nftTokenListUrl()).then((response) => {
+        response.json().then((data) => {
```

Comment:
Shouldn't we handle error here? Also, won't the app will crash if the request was successful but returned invalid data?

Reply:
Sure! But as we don't have the real urls yet, i don't think we have to care about it now. And probably the only thing we can do on errors is log.

The side effect is just the `state` not being changed. So nothing happens, no media showing for the token if it have any.

Reply:
Ok but what I mean is that if `data` (returned from the API) is not an array, `data.find` will throw an error that is not being handled and the explorer screen will be white, right?

Reply:
Will not crash, just throw an exception on console. Anyway, it's protected from crash now: https://github.com/HathorNetwork/hathor-explorer/pull/54/commits/6a8ce3f72b531d5792e9966e7cc57b41b71fb103#diff-b6137568dc9d2ef5b53c1a08a45e9d9ee38a212a149533ce384c5858fc1103d8R81

Reply:
I'm afraid we won't use this method anymore. Take a look at my other comment, please.